download_knesset_recording.py

```diff
@@ -815,12 +815,34 @@ def get_html_transcript(
         for page_idx in range(page_count):
             print(f"Downloading transcript HTML content - page {page_idx + 1}")
 
-            page_content_response = requests.get(
-                f"{knesset_protocol_page_content_api_url}?sProtocolNum={plenum_recording_id}&pageNum={page_idx}",
-                headers=knesset_http_headers,
-            )
-            page_content = json.loads(page_content_response.text)
-            pages_content.append(page_content)
+            can_move_on = False
```

Comment:
Probably better to use backoff.retry:

https://github.com/yairl/r1xbot/blob/5323f61a4f77b0de20b3801537e9c10d84ba93b3/src/services/open_ai/query_openai.py#L104

Reply:
This is awesome - Done.